documentation/element-api/client-server-rpc.asciidoc

```diff
@@ -98,6 +98,12 @@ In client-side Polymer template code, `this` refers to the corresponding element
 
 You can use it anywhere in your client-side Polymer class implementation, and can pass your own arguments in the method.
 Note that the types should match the method declaration on the server side.
+The supported argument types are:
+- `boolean` , `int`, `double`, their boxed types (`Boolean` , etc)  
+- `String`
+- `JsonValue`
+- Enumeration type which is addressed via a string value from the client side JavaScript
+- TemplateModel property types
```

Comment:
TemplateModel is a class so `TemplateModel`. This was a bit cryptic to me, could be helped with link to docs about template model.